server/src/serializers/ReviewSerializer.js

```diff
@@ -0,0 +1,16 @@
+class ReviewSerializer {
+    static async showDetails(review) {
+        const allowedAttributes = ["id", "content", "rating", "spoilerWarning", "movieId", "userId"]
```

Comment:
movieId and userId should be removed as you are not displaying those properties on the front end (the exception to this rule would be the primary `id). Especially the `userId` should not be serialized because we do not case about that information. We are most interested in the entire associated `user` object that is queried on the lines below